site/content/en/docs/Guides/feature-stages.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Feature Stages"
+linkTitle: "Feature Stages"
+date: 2019-09-26T01:20:41Z
```

Comment:
This doesn't have a publish date but I'm not seeing it in the website preview: https://a96d2ff-dot-preview-dot-agones-images.appspot.com/site/docs/

Any idea why it isn't showing up?

Reply:
On recent master after adding this file and running locally with `make site-server` the file got displayed as it should, refer to the screen:

<img width="206" alt="Screenshot 2019-10-01 at 17 09 43" src="https://user-images.githubusercontent.com/5626814/65969956-55e33580-e46e-11e9-9b7c-6deb65d2e9a9.png">

Reply:
It's possible it got deleted, as we only keep the past 30 versions.

Reply:
The preview was there, just not the new page (it's easier to review site changes looking at the actual site rather than the raw md). I see it now though -- not sure what happened before.